src/mqtt.c

```diff
@@ -238,6 +238,20 @@ static void on_message(
   sfree(vl.values);
 } /* void on_message */
 
+static int mqtt_subscribe(mqtt_client_conf_t *conf) {
+  int status;
+
+  status = mosquitto_subscribe(conf->mosq, /* message_id = */ NULL, conf->topic,
```

Comment:
Declare and initialize`status` simultaneously.

```c
int status = mosquitto_subscribe(…
```

https://collectd.org/review-comments#initialize-variables